### DISCUSSION TITLE
过期：懒人包&简易包 2022.0721-0730

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#155] [回收]
## 版本专帖

当前版本 mvtools-std.vpy 存在兼容性问题，已在主分支修复，如有需求则自行拉取替换

<hr /><em>link <a href='https://github.com/hooke007/MPV_lazy/releases/tag/20220721'>mpv-lazy-20220721</a></em>
<hr /><em>link <a href='https://github.com/hooke007/mpv.net_CM/releases/tag/6031'>mpvnet-CM-20220730</a></em>

**Comment by hooke007** [comment]
当期简易包已发布

**Comment by linger040831** [comment]
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/84896599/183078107-0a149340-7d42-43ed-9484-86da8de929b3.png">
<img width="1280" alt="image" src="https://user-images.githubusercontent.com/84896599/183078141-521946c5-0509-466c-a01e-d03f28a006e1.png">
老哥，是哪一条参数控制osc下面显示的参数内容的，mpv-lazy关掉第三方的osc后显示的内容比原版多几项，而且有显示不完全。

**Reply by hooke007** [reply]
原版也被我改掉了，但是一直不用忘了改回去了

https://github.com/hooke007/MPV_lazy/blob/36955cd02074efac4b9a777e3ad2ce369560e980/portable_config/script-opts/osc_lazy.conf#L17-L22

**Comment by hooke007** [comment]
由于操作不当，硬盘数据丢了。其中涉及到的该项目相关的文档部分源码丢了。文档站可能会重建，也可能摆烂。

**Comment by hooke007** [comment]
shinchiro的这个64位版本只兼容微架构等级为 x86-64 v3 的处理器https://github.com/shinchiro/mpv-winbuild-cmake/releases/tag/20220821
<s>后续发布可能会更改待观察</s> 新包命名格式 `mpv-x86_64-v3-xxxxxxxx-git-xxxxxxx.7z`

下次懒人包打包会预置不兼容旧硬件的新构建。

**Reply by skiqing** [reply]
啥时候更新，等到花儿都谢了

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/wiki#%E5%8F%91%E7%89%88%E5%91%A8%E6%9C%9F
